index.html

```diff
@@ -197,81 +200,65 @@ <h3 class="works__project-title">Multi-Post Stories</h3>
+            contact me.
+          </p>
+          <h2 class="h2-span">LET’S CONNECT</h2>
+          <section class="headline__social-media">
+            <ul class="social-media__ul">
+              <li>
+                <a href="https://twitter.com/Mukhammadrizo5" rel="noreferrer noopener" target="_blank">
+                  <img src="images/twitter_icon.svg" alt="twitter logo" />
+                </a>
+              </li>
               <li>
-                <div>Langguages</div>
-                <ul>
-                  <li class="elements">
-                    <img src="images/js.png" alt="java script logo" />
-                    JavaScript
-                  </li>
-                  <li class="elements">
-                    <img src="images/html.png" alt="html logo" />
-                    HTML
-                  </li>
-                  <li class="elements">
-                    <img src="images/css.png" alt="css logo" />
-                    CSS
-                  </li>
-                </ul>
+                <a href="https://www.linkedin.com/in/mukhammadrizooff" rel="noreferrer noopener" target="_blank">
+                  <img src="images/linkedin_icon.svg" alt="linkedin logo" />
+                </a>
               </li>
               <li>
-                <div>Frameworks</div>
+                <a href="https://medium.com/@mukhammadrizooff" rel="noreferrer noopener" target="_blank">
+                  <img src="images/medium_icon.svg" alt="medium logo" />
+                </a>
               </li>
               <li>
-                <div>Skills</div>
+                <a href="https://github.com/mukhammadrizooff" rel="noreferrer noopener" target="_blank">
+                  <img src="images/github_icon.svg" alt="github logo" />
+                </a>
+              </li>
+              <li>
+                <a href="#"><img src="images/like_icon.svg" alt="angellist logo" /></a>
               </li>
             </ul>
+            <a href="#about-myself" class="works__resume">Get my resume</a>
           </section>
         </section>
+        <section class="about-myself__menu">
+          <ul class="about-myself__ul">
+            <li>
+              <h3 class="about-myself__list-title">Langguages</h3>
+              <ul class="langguages-ul">
+                <li class="lang-tags">
+                  <img src="images/js.png" alt="java script logo" />JavaScript
+                </li>
+                <li class="lang-tags">
+                  <img src="images/html.png" alt="html logo" />HTML
+                </li>
+                <li class="lang-tags">
+                  <img src="images/css.png" alt="css logo" />CSS
+                </li>
+              </ul>
+            </li>
+            <li><h3 class="about-myself__list-title">Frameworks</h3></li>
+            <li><h3 class="about-myself__list-title">Skills</h3></li>
+          </ul>
+        </section>
```

Comment:
- [ ] AWESOME job 👍, everything is working as expected but there is one thing left to be done. Inside about me section after your list titles there should be arrow icon. I'm sure you can fix it easily 😄🍀

**Your design** 

![image](https://user-images.githubusercontent.com/69011963/145095571-3316079f-4a6d-4688-8d79-060e8ac114f9.png) 

**Figma design**

![image](https://user-images.githubusercontent.com/69011963/145095652-3b39fee0-d301-41d4-9477-400f94ed8c56.png)

Reply:
Yes, I can thank you.